ansible/roles/util-cfg-service/templates/apps.properties.j2

```diff
@@ -1,5 +1,6 @@
-# Connection details.
-apps.app.listen-port = {{ default_service_port }}
+# Application details.
+apps.app.listen-port      = {{ default_service_port }}
+apps.app.environment-name = {{ environment_name }}
```

Comment:
We need the environment name to namespace folders in Grouper. This will allow multiple environments to use a single Grouper deployment without colliding.